Meta/build-image-raspberry-pi.sh

```diff
@@ -104,7 +104,7 @@ uart_2ndstage=1
 enable_uart=1
 enable_gic=1
 
-kernel=kernel8.img
```

Comment:
Doesn't the name "kernel8.img" tell the rpi firmware loader to load it in 64-bit mode (?)